app/assets/javascripts/discourse/controllers/back-date.js.es6

```diff
@@ -0,0 +1,21 @@
+import ModalFunctionality from 'discourse/mixins/modal-functionality';
+import ObjectController from 'discourse/controllers/object';
+
+export default ObjectController.extend(ModalFunctionality, {
+    content: {},
+    needs: ["topic"],
+
+
+    _forwardAction(name) {
+        const date = this.get('dateValue');
+        const time = this.get('timeValue');
+        const dateTime = date + ' ' + time ;
+        this.get("controllers.topic").send(name, this.get('model'), dateTime);
```

Comment:
It looks like `_forwardAction` only ever calls `changeTimeStamp`. It doesn't seem like the name needs to be dynamic in this case. In fact, it doesn't seem like it needs to be a function at all. Why not just put all this location in the `backDatePost` action?

Additionally, we use two spaces for indentation here. In some of the files you stuck with two but in others like this one you used 4.